### PR TITLE
Fixed typo & improved the readability of the README file.

### PR DESCRIPTION
**Pull request by faizalzakaria**
## Notes

- There is no .env.example but rather env.example.
- Its a bit hard to follow the README.

### CODE REVIEW

**Comment by taufek** [comment]
Thanks @faizalzakaria for this improvement. I will merge this in after I fixed the travis failure. Seems like there is an update with travis latest image causing the build to fail.

File: README.md [hunk 1]
```diff
@@ -46,13 +47,24 @@ because for Mac, it requires performance boost with Docker Sync. Refer to
 `docker-compose.linux.yml` or `docker-compose.mac.yml` to `docker-compose.override.yml`
 in order to get your `ruby` workspace volumes mapped to your host machine.
 
+<a name="prerequisites"></a>
+### Prerequisites
+
+- Create `dockery` folder.
+
+```bash
+mkdir -p ~/.dockery/data/bundle/2.4.2
```
Comment (taufek) [inline README.md]: Is it necessary to create this folder manually? I thought this should be handle by docker-compose.

Comment (faizalzakaria) [reply README.md]: Apparently it is. This is in Mac.

Comment (taufek) [reply README.md]: Seems like this issue only happens if we use `docker-sync`. I've updated `sync.sh` to include creating the necessary folders. I've also updated readme to notify user to use this script to start `docker-sync` service.

**Comment by taufek** [comment]
@faizalzakaria ,

I've changed a bit the readme based on the below changes:

63a8d40bc320f74ada72c2495d4311917c8f4928

I'm still interested with the improvement you are doing here. Could you do it again based on the latest changes? I'm so sorry for the trouble.

**Comment by faizalzakaria** [comment]
That should be good now @taufek .

I still believe the doc could be a little bit better still. :) . Anyway, that is enough for me to bring this up.

This works well with simple project, need a better version for a more complex system/config.

File: README.md [hunk 2]
```diff
@@ -57,6 +58,17 @@ because for Mac, it requires performance boost with Docker Sync. Refer to
 `docker-compose.linux.yml` or `docker-compose.mac.yml` to `docker-compose.override.yml`
 in order to get your `ruby` workspace volumes mapped to your host machine.
 
+<a name="prerequisites"></a>
```
Comment (taufek) [inline README.md]: I think if you removed this `Pre-requisites` section, I can merge the rest in. What do you think?

Comment (faizalzakaria) [reply README.md]: Actually the main problem was the .env file, the reason i created this. But looks like you already fixed it.

So I guess we don't have to create the folder manually ?

**Comment by taufek** [comment]
Yes. If you use `docker-sync`, it requires the folders that it syncs to
exists. So if you use my `sync.sh` script to start it, it will create the
folder the folders for you.

If you don’t use `docker-sync`, `docker-compose` will create the folders
for you.

**Comment by faizalzakaria** [comment]
I confirmed, I dont need that folder to be created manually.

**Review by taufek — APPROVED** [review]
